fds/cli.py

```diff
@@ -29,14 +29,37 @@
                            action="store_true", default=False)
 parser_commit.add_argument('message', help="commit message")
 
+# push
+parser_push = command_subparser.add_parser(
+    'push',
+    help='push commits to remote git and dvc repository'
+)
+parser_push.add_argument('-gr', '--git-remote', help="git remote name, default 'origin'", default="origin")
+parser_push.add_argument('-dr', '--dvc-remote', help="dvc remote name, default 'origin'", default="origin")
+parser_push.add_argument(
+    'branch',
+    help="A git branch that you want to push to (accepts refspec), defaults to current branch",
+    nargs='?'
+)
+# save
+parser_save = command_subparser.add_parser(
+    'save',
+    help='saves all project files to a new version and pushes them to your remote'
```

Comment:
```suggestion
    help='Saves all project files to a new version and pushes them to your remote. Equivalent to running fds add; fds commit; fds push'
```

Reply:
I'm not sure I want to add this, since we might improve the logic to take more decisions on behalf of the user. If you think it's a bad practice for some reason then ok, but in the long run, I want to do smarter things like identify blocks to perform save and helping the user overcome them (by being opinionated).